vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Editor.Hints
+
+open Microsoft.VisualStudio.FSharp.Editor
+open FSharp.Compiler.CodeAnalysis
+open FSharp.Compiler.Symbols
+open FSharp.Compiler.Text
+open Hints
+
+module InlineParameterNameHints =
+
+    let private getHint (range: range, parameter: FSharpParameter) =
+        {
+            Kind = HintKind.ParameterNameHint
+            Range = range.StartRange
+            Parts = [ TaggedText(TextTag.Text, $"{parameter.DisplayName} = ") ]
+        }
+
+    let private doesParameterNameExist (parameter: FSharpParameter) = 
+        parameter.DisplayName <> ""
+
+    let isValidForHint (symbol: FSharpMemberOrFunctionOrValue) =
+        // is there a better way?
+        let isNotBuiltInOperator = 
+            symbol.DeclaringEntity 
+            |> Option.exists (fun entity -> entity.CompiledName <> "Operators")
+
+        symbol.IsFunction
+        && isNotBuiltInOperator // arguably, hints for those would be rather useless
```

Comment:
I think this is missing a `symbolUse.IsFromUse` check, because without it function definitions are eligible for a hint, even though you will later find no applied arguments, so nothing is shown. I will fix it in the member and ctor PR.

Reply:
Right, so the behavior is by coincidence :) Yes, this will be a valid check, thanks.